docs_user/modules/openstack-pull_openstack_configuration.adoc

```diff
@@ -90,3 +90,65 @@ Once it's done, you should have into your local path a directory per services su
       ▾ glance/
       ▾ keystone/
 ----
+
+== Get services topology specific configuration
+
+Define the shell variables used in the steps below. The values are
+just illustrative, use values that are correct for your environment:
```

Comment:
that's the only way to carry on those after we shut down the source cloud services...
we could use env files, like I [suggested earlier](https://github.com/openstack-k8s-operators/data-plane-adoption/pull/146), which should also help users to interrupt and resume going through the documented steps w/o the need to repeat all the "lost" exports and set commands

Reply:
actually, that's not a simple task, to store those evaluated multi-line values in an env file so that those could be sources w/o problems... I'll try and let you know

Reply:
@bogdando I think it's ok to keep the env vars. Writing to a file would be good for persistence, but it could further inflate the docs, so it may not be a silver bullet. (?)

Reply:
please see https://github.com/openstack-k8s-operators/data-plane-adoption/pull/271